README.md

```diff
@@ -3,11 +3,7 @@
 The purpose of orca is to pull apart large `OPeNDAP` requests to `THREDDS` into bite-sized chunks. These chunks are then reassembled before returning to the user.
 
 ## Installation
-We use `make` to handle the installation process. Copy and paste this section into your terminal:
-```
-make
-source /tmp/orca-venv/bin/activate
-```
+We use `make` and `pipenv` for our installation process so the setup should be quick and painless! Simply paste `make` into your terminal to run a full install. If you wish to run commands from within the virtual environment use `pipenv shell` to activate it.
```

Comment:
Might be worth expanding this a tiny bit: The default target does more than install, it makes targets `test` and `pre-commit-hook` too.

Reply:
Also, maybe it would be a good idea to add some info on `pipenv`, even just a hyperlink to its doc, so that people can know other cool things to do, such as look at the dep tree etc.